src/main/java/com/opentable/util/ExceptionSwallower.java

```diff
@@ -43,8 +47,6 @@ public static Runnable swallowExceptions(Runnable in)
                 LOGGER.error("Uncaught exception swallowed", t);
                 if (t instanceof Error) {
                     throw t;
-                } else if (t instanceof InterruptedException) {
```

Comment:
This condition is impossible.

Reply:
How is that impossible. You catch throwable. And remember people can sneakythrows and stuff like that